t/Equaldex.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+spice is_cached => 1;
+
+ddg_spice_test(
+    [qw( DDG::Spice::Equaldex)],
+
+    'lgbt china' => test_spice(
+        '/js/spice/equaldex/china',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+
+    'lesbian rights cyprus' => test_spice(
+        '/js/spice/equaldex/cyprus',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'gay rights australia' => test_spice(
+        '/js/spice/equaldex/australia',
+        call_type => 'include',
+        caller => 'DDG::Spice::Equaldex'
+    ),
+    
+    'transgender rights usa' => test_spice(
```

Comment:
@moollaza Happy to switch to a country test file or just deploy a regex hack for the `usa` edge case. I'll try explaining the issue a little better.

````perl
return $country if defined country2code($country);
````

I'm using the `country2code` function to test full country names e.g **"Australia", "Brazil"** `country2code` just converts a full country name to the ALPHA code.

`country2code` should return a value and be "defined" if the country name is correct, however it returns a value for the quires `usa` or `us` (which are not full country names)

`country2code("usa")` returns `"usa"`
`country2code("us")` returns `"us"`

This is due to some variant names being added to the `Locate::Country` package. :cry: 

I think the best course of action, provided these are the only quires effected is to simply convert `usa`, `us` to the full country name using regex :+1: